HOT: fix empty samplings (#322)
pyfar/spharpy: HISTORY.rst

History
=======

Fixed

> ^^^^

Author: ```suggestion
^^^^^
```

> - SamplingSphere can now be empty, similar to the pyfar.Coordinates class (PR #322)

Reviewer: ```suggestion
- Creating a SamplingSphere without passing arguments now returns an empty object instead of an object containing NaN values, similar to the pyfar.Coordinates class (PR #322)
```
I think this describes it more precisely.

History
=======

> - SamplingSphere can now be empty, similar to the pyfar.Coordinates class (PR #322)

Author: ```suggestion

Fixed
^^^^

- SamplingSphere can now be empty, similar to the pyfar.Coordinates class (PR #322)
```

> 

Reviewer: Please indicate that this is a bugfix
